### PR TITLE
Expand the contributing some to point to the vscode docs.

### LINKED ISSUE #168: Need a readme how to build the plugin
Last published release of the plugin is vscode-bazel 0.3.0 which seems to be rather old. I would like to build the latest from master and try if it fixes my problem with the debugger not stopping on the breakpoints, but I cannot figure out how to build from sources. Is it possible to extend CONTRIBUTING.md with a little Getting Started how to build it and load to vscode? I see scripts\build.sh, but apparently some preparation needs to be done before it can run.

### PR DESCRIPTION
**Pull request by thomasvl**
Fixes #168

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,6 +27,18 @@ again.
 
 ## Setting up your development environment
 
+To contribute, you likely should already be familiar with VSCode extensions.
```
Comment (allevato) [inline CONTRIBUTING.md]:
```suggestion
To contribute, you likely should already be familiar with VS Code extensions.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -27,6 +27,18 @@ again.
...
+The best place to start is probably their
+[guide](https://code.visualstudio.com/api/get-started/your-first-extension).
+
+Once somewhat familiar with the process, you just need to checked out this
```
Comment (allevato) [inline CONTRIBUTING.md]:
```suggestion
Once somewhat familiar with the process, you just need to check out this
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -27,6 +27,18 @@ again.
...
+project, do an `npm install` to gets the required packages into the local
```
Comment (allevato) [inline CONTRIBUTING.md]:
```suggestion
project, do an `npm install` to get the required packages into the local
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -27,6 +27,18 @@ again.
...
+checkout's *node_modules* and then open the directory in VSCode. There are
```
Comment (allevato) [inline CONTRIBUTING.md]:
```suggestion
checkout's *node_modules* and then open the directory in VS Code. There are
```

**Comment by thomasvl** [comment]
Suggestions accepted.

**Pull request merged**